samples/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -10,78 +10,367 @@
 #define MODULE ble_bond
 #include "module_state_event.h"
 #include "button_event.h"
+#include "ble_event.h"
 
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_BOND_LOG_LEVEL);
 
-static bool bonds_initialized;
-static bool remove;
 
-static int bonds_remove(void)
+#define SHORT_CLICK_MAX 500 /* ms */
```

Comment:
`K_MSEC(500)` and you can remove comment

Reply:
Fixed